setup.php

```diff
@@ -1289,14 +1289,23 @@ function cloneClicked() {
                         }
 
                         $short_tag = ini_get('short_open_tag') ? 'On' : 'Off';
+                        $short_tag_style = strcmp($short_tag, 'Off') === 0 ? '' : 'text-danger';
```

Comment:
to make these statements more readable, lets place parentheses around the conditionals:
```
$short_tag_style = (strcmp($short_tag, 'Off') === 0) ? '' : 'text-danger';
```

Reply:
ok, updated commit,